lectures/long_run_growth.md

```diff
@@ -4,41 +4,40 @@ jupytext:
     extension: .md
     format_name: myst
     format_version: 0.13
-    jupytext_version: 1.15.2
+    jupytext_version: 1.16.1
 kernelspec:
   display_name: Python 3 (ipykernel)
   language: python
   name: python3
 ---
 
-
-# Economic Growth Evidence
+# Long-run Growth
 
 ## Overview
 
-In this lecture we use Python, Pandas,    and Matplotlib to download, organize, and visualize historical data on GDP growth.
+In this lecture we use Python, {doc}`pandas<pyprog:pandas>`, and {doc}`Matplotlib<pyprog:matplotlib>` to download, organize, and visualize historical data on economic growth.
 
-In addition to learning how to deploy these tools more generally, we'll use them to describe facts about economic growth experiences across many countries  over several  centuries.
+In addition to learning how to deploy these tools more generally, we'll use them to describe facts about economic growth experiences across many countries over several centuries.
 
-Such "growth facts" are  interesting for a variety of reasons.  
+Such "growth facts" are interesting for a variety of reasons.  
 
-Explaining growth facts is a principal purpose of both "development economics" and "economic history".
+Explaining growth facts is a principal purpose of both "development economics" and "economic history".
 
-And  growth facts are important inputs into historians'  studies of  geopolitical forces and dynamics.
+And growth facts are important inputs into historians' studies of geopolitical forces and dynamics.
 
 
-Thus, Adam Tooze's account of the geopolitical precedents and antecedents of World War I begins by describing how Gross National Products of European Great Powers had evolved during the 70 years preceding 1914 (see chapter 1 of {cite}`Tooze_2014`).
+Thus, Adam Tooze's account of the geopolitical precedents and antecedents of World War I begins by describing how the Gross Domestic Products (GDP) of European Great Powers had evolved during the 70 years preceding 1914 (see chapter 1 of {cite}`Tooze_2014`).
 
-Using the very same data that Tooze used to construct his figure, here is our version of his chapter 1 figure.
+Using the very same data that Tooze used to construct his figure (with a slightly longer timeline), here is our version of his chapter 1 figure.
 
 
 ```{figure} _static/lecture_specific/long_run_growth/tooze_ch1_graph.png
-:width: 75%
+:width: 80%
```

Comment:
@HumphreyYang could you post a screenshot of the new size? Maybe we want it larger than 80% give it was `75%` already